doc/conf.py

```diff
@@ -163,4 +178,4 @@ def resolve_intersphinx_aliases(app, env, node, contnode):
 def setup(app):
     app.connect('doctree-read', resolve_internal_aliases)
     app.connect('missing-reference', resolve_intersphinx_aliases)
-    app.connect('autodoc-skip-member', autodoc_skip_member_handler)
+    app.connect('autodoc-skip-member', autodoc_skip_member_handler)
```

Comment:
Can you add the newline at the end back? I think it's usually a best-practice to always end files with a newline